mlir/utils/jenkins/build-dockerfile.sh

```diff
@@ -20,25 +20,29 @@ start_clean_docker() {
 
 # Build the docker image and push it to the repository
 build_rocm_image() {
-  if docker build -t rocm-mlir -f ./mlir/utils/jenkins/Dockerfile ./mlir/utils/jenkins ; then
```

Comment:
Is my bash knowledge failing me here or how did this ever work ? 
From what I know if a command succeeded. 

Just to be sure did some lookarounds as well : https://unix.stackexchange.com/questions/22726/how-to-conditionally-do-something-if-a-command-succeeded-or-failed

Reply:
... yeah, this is exactly backwards, good catch!